### PR TITLE
(retro) c8 backup and restore review

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

Based on changes made in https://github.com/camunda/camunda-platform-docs/pull/1350. Note that this is a **retroactive review made on an already merged PR** given I was out on FTO at the time of the original request for review.

### CODE REVIEW

File: docs/self-managed/backup-restore/backup-and-restore.md [hunk 1]
```diff
@@ -39,20 +43,17 @@ PUT /_snapshot/my_repository/camunda_zeebe_records_backup_x
...
-
-9. Take Backup `x` of Zeebe. (See [How to take Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md))
-10. Wait until Backup `x` of exported zeebe records is complete.
-11. Wait until Backup `x` of Zeebe is completed. (See [How to monitor Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md))
-12. Resume exporting in Zeebe. (See [Zeebe management api](/self-managed/zeebe-deployment/operations/management-api.md))
+By default, the indices are prefixed with `zeebe-record`. If you have configured a different prefix when configuring Elasticsearch exporter in Zeebe, use this instead. 9. Take a backup `x` of Zeebe. See [how to take a Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md). Wait until the backup `x` of the exported Zeebe records is complete and wait until the backup `x` of Zeebe is completed before proceeding. See [how to monitor a Zeebe backup](self-managed/backup-restore/zeebe-backup-and-restore.md). 10. Resume exporting in Zeebe. See [Zeebe management API](/self-managed/zeebe-deployment/operations/management-api.md).
```
Comment (deepthidevaki) [inline docs/self-managed/backup-restore/backup-and-restore.md:46]: Steps 9-12 should be in bullet points.

**Review by deepthidevaki — CHANGES_REQUESTED** [review]
Thank you for going over the doc and improving it. Just have one minor comment on formatting.

File: docs/self-managed/backup-restore/backup-and-restore.md (hunk 1 above)
Comment (christinaausley) [reply docs/self-managed/backup-restore/backup-and-restore.md]: Nice catch on the typo 👍 Adjusted in the latest commit.

**Review by akeller — CHANGES_REQUESTED** [review]
This PR should also include changes for 8.1. Can you add them?

**Review by akeller — APPROVED** [review]

**Pull request merged**